fossunited/chapters/doctype/foss_event_rsvp_submission/foss_event_rsvp_submission.py

```
@@ -44,9 +48,20 @@ def after_insert(self):
 
     def before_save(self):
-        if self.has_value_changed("status") and not self.is_new():
+        if self.has_value_changed("subscribe_chapter_mailing") or self.has_value_changed("status"):
             self.handle_add_to_email_group()
 
+    def on_trash(self):
+        try:
+            # remove from mailing group as well
+            self.subscribe_chapter_mailing = 0
+            self.handle_add_to_email_group()
+        except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)